### PR TITLE
Enable some ignored tests

### PR DESCRIPTION
**Pull request by siy**
Signed-off-by: Sergiy Yevtushenko <sergiy.yevtushenko@pragmaticcoders.com>

### CODE REVIEW

File: txscript/reference_test.go [hunk 1]
```diff
@@ -186,7 +186,7 @@ func createSpendingTx(sigScript, pkScript []byte) *wire.MsgTx {
 // as expected.
 //TODO: enable test and make it working
 func TestScriptInvalidTests(t *testing.T) {
-	//For some reason this test hangs IDE and PC
+	//Depends on enternal files with binary data (encoded)
```
Comment (jksf) [inline txscript/reference_test.go]: Typo: external

Comment (jksf) [reply txscript/reference_test.go]: Also please leave space after comment symbol like
> `//Depends`
to
> `// Depends`

**Comment by siy** [comment]
Fix applied

**Review by jksf — APPROVED** [review]
LGTM

**Pull request merged**